src/test/scala/uk/gov/hmrc/test/ui/driver/BrowserDriver.scala

```diff
@@ -33,14 +33,16 @@ trait BrowserDriver extends LazyLogging {
   if (browser.isEmpty)
     sys.props += ("browser" -> "chrome")
 
-  private val options = browser match {
-    case Some("chrome") | None => Some(new ChromeOptions().addArguments("--remote-allow-origins=*"))
-    case Some("firefox")       => None
-    case Some(_)               => None
-  }
+  private val options =
+    browser match {
+      case Some("firefox") => None
+      case _               => Some(new ChromeOptions().addArguments("--remote-allow-origins=*"))
```

Comment:
Does ChromeOptions() work with non-Chrome browsers?

Reply:
we have separate case for Firefox